metrics/mir_team_bugs.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+"""Submit metrics for MIR Team queue sizes.
+
+Copyright 2019 Canonical Ltd.
+Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
+
+Based on triage.py:
+Joshua Powers <josh.powers@canonical.com>
+"""
+import argparse
+
+from metrics.helpers import lp
+from metrics.helpers import util
+
+
+def collect(dryrun=False):
+    """Submit data to Push Gateway."""
+    unassigned = lp.get_team_unassigned_bugs(team='ubuntu-mir',
+                                             distro='Ubuntu')
+    incomplete = lp.get_team_incomplete_bugs(team='ubuntu-mir',
+                                             distro='Ubuntu')
+    pending = lp.get_mir_in_review()
+    security = lp.get_mir_in_security_review()
+    approved = lp.get_approved_mirs()
+
+    print('Unassigned Total: %s' % unassigned)
```

Comment:
IIRC most of the scripts only print if dryrun is True.

Reply:
Well, they won't push if it's dryrun, but otherwise what's the point of doing the dryrun at all? I would see this as "let me see, get the data and show it to me but don't put it in the database".